next-semver/main.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 VMware, Inc.
```

Comment:
There is already tooling for this, I'd not build our own i.e https://github.com/fsaintjacques/semver-tool